com.unity.shadergraph/Documentation~/Texel-Size-Node.md

```diff
@@ -13,8 +13,10 @@ If you experience texture sampling errors while using this node in a graph which
 | Name        | Direction           | Type  | Binding | Description |
 |:------------ |:-------------|:-----|:---|:---|
 | Texture      | Input | Texture | None | Texture asset |
-| Width      | Output | Float    | None | Texel width |
-| Height | Output      |    Float    | None | Texel height |
+| Width      | Output | Float    | None | Texture width |
```

Comment:
should update the description to note the units returned:    Texture width in texels

Reply:
Made this change in the new version of the documentation that Sarah uploaded.

Reply:
Made this change in the new version of the documentation that Sarah uploaded.